### PR TITLE
Promote @TylerHelmuth to maintainer

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Approvers ([@open-telemetry/collector-contrib-approvers](https://github.com/orgs
 - [David Ashpole](https://github.com/dashpole), Google
 - [Evan Bradley](https://github.com/evan-bradley), Dynatrace
 - [Sean Marciniak](https://github.com/MovieStoreGuy), Atlassian
-- [Tyler Helmuth](https://github.com/TylerHelmuth), Honeycomb
+- [Ziqi Zhao](https://github.com/fatsheep9146), Alibaba
```
Comment (jpkrohling) [inline README.md:95]: Perhaps this needs a rebase?

Comment (djaglowski) [inline README.md:95]:
```suggestion
```

Carryover from #18310

**Review by jpkrohling — COMMENTED** [review]
+1 on having @TylerHelmuth as maintainer, but apparently this PR might need a rebase?

**Review by codeboten — COMMENTED** [review]
👍🏻 thumbs up on @TylerHelmuth and on rebase :D

**Review by codeboten — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]
Thanks everyone!  Being a part of this community has been so fun and rewarding and I'm excited to continue that work 🚀

**Review by dmitryax — APPROVED** [review]

**Review by jpkrohling — APPROVED** [review]

**Comment by lizthegrey** [comment]
congrats @codeboten!

**Comment by codeboten** [comment]
> congrats @codeboten!

@lizthegrey thanks! but I'll assume you meant @TylerHelmuth :D

**Comment by lizthegrey** [comment]
> > congrats @codeboten!
> 
> @lizthegrey thanks! but I'll assume you meant @TylerHelmuth :D

I clearly have not had enough coffee today to be cut pasting names around, edited.

**Pull request merged**